paths/reports/locales/index.yaml

```diff
@@ -0,0 +1,57 @@
+---
+summary: List Locale Reports
+description: List all locale reports for the given project
+operationId: reports/locales/list
+tags:
+  - Reports
+parameters:
+  - "$ref": "../../../parameters.yaml#/X-PhraseApp-OTP"
+  - "$ref": "../../../parameters.yaml#/project_id"
+  - "$ref": "../../../parameters.yaml#/page"
+  - "$ref": "../../../parameters.yaml#/per_page"
+  - "$ref": "../../../parameters.yaml#/locale_codes"
+  - "$ref": "../../../parameters.yaml#/tag"
```

Comment:
only one tag can be specified? also, how are locales filtered by tag? or is it for filtering keys in the reports?

Reply:
The second. It is filtering keys by given tag. And sice the whole service only supports single tag I did not want to refactor this :slightly_smiling_face: